src/main/java/com/googleinterns/zoomtube/servlets/LectureServlet.java

```diff
@@ -94,7 +94,13 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
     String lectureName = request.getParameter(PARAM_NAME);
     Entity lectureEntity = LectureUtil.createEntity(lectureName, videoUrl, videoId.get());
     datastore.put(lectureEntity);
-    initializeTranscript(lectureEntity);
+    try {
+      initializeTranscript(lectureEntity);
+    } catch (IOException | ServletException e) {
```

Comment:
nit: `e` can be more descriptive in case it is used in the future.

Reply:
nah, i think it can be changed if it's ever needed, but for now i don't want to draw attention to it